Docs/svt-hevc_encoder_user_guide.md

```diff
@@ -267,40 +271,40 @@ The encoder parameters present in the Sample.cfg file are listed in this table b
 | **SourceWidth** | -w | [64 - 8192] | 0 | Input source width |
 | **SourceHeight** | -h | [64 - 4320] | 0 | Input source height |
 | **FrameToBeEncoded** | -n | [0 - 2^31 -1] | 0 | Number of frames to be encoded, if number of frames is > number of frames in file, the encoder will loop to the beginning and continue the encode. 0 encodes the full clip. |
-| **BufferedInput** | -nb | [-1, 1 to 2^31 -1] | -1 | number of frames to preload to the RAM before the start of the encode. If -nb = 100 and –n 1000 --> the encoder will encode the first 100 frames of the video 10 times. Use -1 to not preload any frames. |
+| **BufferedInput** | -nb | [-1, 1 to 2^31 -1] | -1 | number of frames to preload to the RAM before the start of the encode. If -nb = 100 and –n 1000 --> the encoder will encode the first 100 frames of the video 10 times. Use -1 to not preload any frames. Note: 0 is not valid input. This parameter is best used to measure the impact of disk reading on encoding speed and is most noticeable when frames sizes are 4k or 8k.  Because frames are repeated when value specified is less than the frame count, you should expect bitstreams to be different. |
```

Comment:
Thanks for the change.
1. -nb 0, doesn't need to be mentioned.
2. measure -> eliminate, do you think it is better?
3. add (-n) after "frame count".

Reply:
yes agree